src/Core/src/WindowOverlay/WindowOverlay.cs

```diff
@@ -69,7 +69,7 @@ public bool IsVisible
 		}
 
 		/// <inheritdoc/>
-		public float Density { get; internal set; } = 1;
+		public float Density => Window?.RequestDisplayDensity() ?? 0f;
```

Comment:
1f would be a better fallback value than 0f

Reply:
Do you want the likely wrong value in the fallback or a value you know indicates it’s incorrect and you can choose 1f if you want in that case?

Reply:
We'll have to fall back to 1 anyway:) Given that a) this is public API and b) this value is likely to be used in calcs like uitsPerPixel = 1/Density then returning 0 is likely to cause divided-by-0.

But if you believe 0 is better than 1 then I can live with that :)

Reply:
I agree that density should be 1 because some platforms do not support this, so will always return 1. The only reason for the null check if there is no window, so it is better to be consistent and just return 1.